### DISCUSSION TITLE
Arabic Language

### DISCUSSION THREAD

**Discussion by abdullasuliman** [SingularityLabs-ai/beyond-bard-mini#5] [General]
Thank you for this beautiful addition to the Chrome browser. The Arabic language is not added to the list of languages in the extension settings.
I added the Arabic language by editing the add-on, but I noticed that part of the YouTube summary is in Chinese.

I hope to review, correct, and add the Arabic language.

Greetings and thanks to you

**Comment by ishandutta2007** [comment]
If you meant you updated this current open-source codebase then You can send us a pull requests with your changes , we will be happy to add you to contributors, even though the current github is little older version but we can patch your changes on the newer version too.
